### PR TITLE
SRVCOM-1288 + SRVCOM-1664: Clean up attributes, prereqs

### PR DESCRIPTION
**Pull request by abrennan89**
Jiras: https://issues.redhat.com/browse/SRVCOM-1288 + https://issues.redhat.com/browse/SRVCOM-1664

Applies for OCP 4.6+

Direct preview links:
- https://deploy-preview-41780--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-using-brokers.html
- https://deploy-preview-41780--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-triggers.html
- https://deploy-preview-41780--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-kafka-developer.html

### CODE REVIEW

File: modules/serverless-create-trigger-odc.adoc [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
 . Select your sink as a *Subscriber* from the drop-down list.
-+
 . Click *Add*.
 
 .Verification
-* After the subscription has been created, it is represented as a line that connects the broker to the service in the *Topology* view:
-+
-image::verify-trigger-odc.png[Trigger in the Topology view]
+
+* After the subscription has been created, you can view it represented as a line that connects the broker to the event sink in the *Topology* view.
```
Comment (bobfuru) [inline modules/serverless-create-trigger-odc.adoc:28]: If it doesn't change the meaning, you might consider front-loading the object they are acting on (Topology view): 
```suggestion
* After the subscription has been created, you can view it in the *Topology* view, where it is represented as a line that connects the broker to the event sink.
```

File: modules/serverless-creating-broker-labeling.adoc [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
 //
-// * serverless/knative_eventing/serverless-using-brokers.adoc
+// * /serverless/develop/serverless-using-brokers.adoc
 
 :_content-type: PROCEDURE
 [id="serverless-creating-broker-labeling_{context}"]
 = Creating a broker by labeling a namespace
 
-You can create the `default` broker automatically by labeling a namespace that you own or have write permissions for.
+You can create the `default` broker automatically by labelling a namespace that you own or have write permissions for.
```
Comment (bobfuru) [inline modules/serverless-creating-broker-labeling.adoc:9]: You brought a bit o' the UK over to the US, eh? 😛

File: modules/serverless-deleting-broker-injection.adoc [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 //
-// * serverless/knative_eventing/serverless-using-brokers.adoc
+// * /serverless/develop/serverless-using-brokers.adoc
 
 :_content-type: PROCEDURE
 [id="serverless-deleting-broker-injection_{context}"]
 = Deleting a broker that was created by injection
 
-Brokers created by injection, by using a namespace label or trigger annotation, are not deleted permanently if a developer removes the label or annotation. You must manually delete these brokers.
+Brokers created by injection, by using a namespace label or trigger annotation, are not deleted permanently if you remove the label or annotation. You must manually delete these brokers.
```
Comment (bobfuru) [inline modules/serverless-deleting-broker-injection.adoc:9]: This first sentence is a bit hard to follow. Perhaps consider using active voice? You might also consider breaking it up into two sentences. I gave it a go but couldn't come up with something worth sharing. 😁

**Review by bobfuru — COMMENTED** [review]
A couple of thoughts for consideration but overall LGTM!

File: modules/serverless-creating-broker-labeling.adoc (hunk 2 above)
Comment (abrennan89) [reply modules/serverless-creating-broker-labeling.adoc:9]: lol I brought my laptop when I moved from Ireland so I guess the spellcheck is still UK English 😂 I'll switch it back, thanks!

File: modules/serverless-create-trigger-odc.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/serverless-create-trigger-odc.adoc:28]: Great suggestion, thank you!

File: modules/serverless-deleting-broker-injection.adoc (hunk 3 above)
Comment (abrennan89) [reply modules/serverless-deleting-broker-injection.adoc:9]: Had a go at rewording this, lmk what you think 🙂

File: modules/serverless-creating-broker-labeling.adoc (hunk 2 above)
Comment (bobfuru) [reply modules/serverless-creating-broker-labeling.adoc]: It's fine either way!

**Review by bobfuru — COMMENTED** [review]
Looks stupendous to me!
